charts/tempo/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo
 description: Grafana Tempo Single Binary Mode
 type: application
-version: 1.7.1
+version: 1.7.2
```

I think feature should bump to minor version:

```suggestion
version: 1.8.0
```